eZ/Publish/Core/Repository/User/PasswordHashServiceInterface.php

```diff
@@ -8,13 +8,14 @@
 
 namespace eZ\Publish\Core\Repository\User;
 
-/**
- * @internal
- */
```

Comment:
If you don't want it internal, I'd prefer to see it in API.

Reply:
To prevent BC we could `class_alias` it in API repository / namespace to occupy both class names until 4.0?

Reply:
Done in e9c6d71. I've marked PasswordHashServiceInterface as deprecated in v3.3 and change it's definition to extend PasswordHashService from PAPI. This should prevent any BC breaks.